src/scripts/batch/hit_counts.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Using user query strings from the input stream,
+ * this script gets the number pathways and interactions (SIF edges).
+ * Results will be the same as or close (see comments below) to
+ * what one could get at apps.pathwaycommons.org if settings and queries are same.
+ *
+ * This script simply calls the app-ui server-side modules
+ * (does not start the nore server nor web app),
+ * which, in order, search for pathways and inferred simple
+ * interactions in the Pathway Commons backend (PC2 and Sifgraph services).
+ *
+ * @author Igor Rodchenkov
+ */
+
+const {getInteractionGraphFromPC} = require('../../server/routes/interactions/generate-interactions-json');
+const {search} = require('../../server/routes/search/search');
+const csv = require('csv');
+const {pipeline} = require('stream');
+const logger = require('../../server/logger');
+
+
+const transform = csv.transform((data, callback) => {
+  let out = [];
+  let q = data['Term'];
+  out.push(q);
+
+  // apply the same hack as done on the app-ui's client-side, in ServerAPI.search (before  going /api/search)
+  if (/^((uniprot|hgnc):\w+|ncbi:[0-9]+)$/i.test(q)) {
+    q=q.replace(/^(uniprot|ncbi|hgnc):/i,"");
+  }
+
```

Comment:
Yeah I should probably ditch this in the client as we don't use it anymore.

Reply:
Why.. this was in the client to support linking from UniProt (eventually). Let it be. I copied this here because the script is suppose to work alike the UI, and the Reactome users did type something like "uniprot:q12345", which would not match anything in our app without the above code, i think.